src/main/java/mate/academy/springboot/datajpa/dto/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+public interface DtoMapper<M, R, S> {
+    M toModel(R requestDto);
+
+    S toResponseDto(M model);
+}
```

Comment:
Better divide this mapper for request and response DTO, this combination of generics made code poorly readable